doc/src/site/sphinx/tutorials/sw_xgboost.rst

```diff
@@ -77,6 +82,12 @@ Train the model. You can configure all the available XGBoost arguments using pro
     model = estimator.fit(spark_frame)
 
 
+You can also get raw model details by calling the *getModelDetails()* method available on the estimator as:
+
+.. code:: python
+
+    print(model.getModelDetails())
```

Comment:
Scala exemple doesn't print details to console. It would be nice if it was consistent.

Reply:
Agree! The issue here is that Python repl prints multi-line strings on one line with `\n` in it. So when we do:

`details = model.getModelDetails()` it is printed that ugly, but in Scala just fine. One way is to change method to `printModelDetails` however it does not sound right to me.

Any ideas please?